common/src/main/java/org/opensearch/ml/common/parameter/MLPredictionOutput.java

```diff
@@ -82,7 +82,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (taskId != null) {
             builder.field(TASK_ID_FIELD, taskId);
         }
-        if (status != null){
+        if (status != null) {
```

Comment:
btw: we should have checkstyle, lincense checks, etc for this repo.

Reply:
yes, totally agree, create an issue to improve checkstyle